fuel/app/classes/controller/api/admin.php

```diff
@@ -64,4 +68,37 @@ public function post_user($user_id)
 		unset($user->id);
 		return \Service_User::update_user($user_id, $user);
 	}
+
+	public function get_widget_search($input)
```

Comment:
need to test this when $input isn't set, but it'd be nice to add types
```suggestion
	public function get_widget_search(string $input)
```